src/hotspot/os/windows/globals_windows.hpp

```diff
@@ -38,6 +38,10 @@
 product(bool, UseAllWindowsProcessorGroups, false,                        \
         "Use all processor groups on supported Windows versions")         \
                                                                           \
+product(bool, EnableAllLargePageSizesForWindows, false, EXPERIMENTAL,     \
```

Comment:
Should we rename this flag to something like EnableMultipleWindowsLargePageSizes? The flag suggests that we can enable all large page sizes but we reject values that aren't a multiple of min page size, for example.

Reply:
agreed with Saint to keep the name.